lib/config/presets/internal/default.ts

```diff
@@ -586,6 +586,12 @@ export const presets: Record<string, Preset> = {
     separateMajorMinor: true,
     separateMultipleMajor: true,
   },
+  separateMultipleMinorReleases: {
+    description:
+      'Separate each `minor` version of dependencies into individual branches/PRs.',
+    separateMinorPatch: true,
+    separateMultipleMinor: true,
```

Comment:
Shouldn't setting separateMultipleMinor=true implicitly enable separateMinorPatch=true? I can't think of a scenario where it makes sense to skip the patch update